base/applications/games/winmine/lang/zh-CN.rc

```diff
@@ -2,7 +2,7 @@ LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
 
 STRINGTABLE
 BEGIN
-    IDS_APPNAME "Wine地雷"
+    IDS_APPNAME "Wine 扫雷"
```

Comment:
Shouldn't these changes be sent upstream to Wine? :thinking:

Reply:
I'll contact with Wine's translator about this problem.